app/views/presentations/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Presentations</h1>
+
+<%= link_to 'New Presentation', new_presentation_path %>
+
+<table>
+  <thead>
+    <tr>
+      <th>Presentation</th>
+      <th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @presentation.each do |presentation| %>
+      <tr>
+        <td><%= presentation.topic %></td>
+        <td>
+          <%= link_to 'Show', presentation_path(presentation) %>
+          <%= link_to 'Edit', edit_presentation_path(presentation) %>
+          <%= link_to 'Delete', presentation_path(presentation),
+                      method: :delete,
+                      data: { confirm: 'Are you sure?' } %>
```

Comment:
yeah, I'm ok with this. Alternatively we only have links to show and destroy here, and edit is linked to from the show page.